src/frontend/screens/Settings/ProjectSettings/index.tsx

```diff
@@ -105,6 +126,19 @@ export const ProjectSettings = () => {
           onPress={() => navigate('YourTeam')}
         />
       )}
+      {isCoordinator && (
+        <SettingsCardRow
+          icon={<ExchangeIcon width={24} height={24} />}
+          title={formatMessage(
+            remoteArchiveOn ? m.remoteArchiveOn : m.remoteArchiveOff,
+          )}
+          subtitle={formatMessage(m.remoteArchiveDesc)}
+          buttonText={formatMessage(m.viewDetails)}
+          onPress={() =>
+            navigate(remoteArchiveOn ? 'RemoteArchiveOn' : 'RemoteArchiveOff')
+          }
+        />
+      )}
```

Comment:
Upon some additional thinking, one thing that isn't clear to me from a UX perspective is if this screen should only be able to be accessed by a coordinator.

My initial thought is that anyone should be able to view this screen, but the actions that can be taken within it should be restricted (that screen currently does that). If that's the desired UX, this conditional check here is misaligned with that.